pkg/kapp/cmd/app/deploy.go

```diff
@@ -405,6 +405,11 @@ func (o *DeployOptions) calculateAndPresentChanges(existingResources,
 		changeFactory := ctldiff.NewChangeFactory(conf.RebaseMods(), conf.DiffAgainstLastAppliedFieldExclusionMods())
 		changeSetFactory := ctldiff.NewChangeSetFactory(o.DiffFlags.ChangeSetOpts, changeFactory)
 
+		err := ctldiff.NewChangeSetWithPeriodicRs(existingResources, newResources).Calculate()
```

Comment:
`NewChangeSetWithPeriodicRs.Calculate()` seems to indicate that we are calculating changes 🤔
How about we something like `RenewableResources` and `Prepare()`? (Just a thought, we could use something else)